wkcuber/wkcuber/convert.py

```diff
@@ -0,0 +1,89 @@
+"""This module converts an image stack to a WEBKNOSSOS dataset."""
+
+from argparse import Namespace
+from multiprocessing import cpu_count
+from pathlib import Path
+from typing import Optional
+
+import typer
+from rich import print as rprint
+from typing_extensions import Annotated
+
+from webknossos import Dataset
+from webknossos.utils import get_executor_for_args
+from wkcuber.utils import DataFormat, DistributionStrategy, VoxelSize, parse_voxel_size
+
+app = typer.Typer(invoke_without_command=True)
+
+
+@app.callback()
+def main(
+    *,
+    source: Annotated[Path, typer.Argument(help="Path to your image data.")],
+    target: Annotated[
+        Path, typer.Argument(help="Target path to save your WEBKNOSSOS dataset.")
+    ],
+    voxel_size: Annotated[
+        VoxelSize,
+        typer.Option(
+            help="The size of one voxel in image data.",
+            parser=parse_voxel_size,
+        ),
+    ],
+    data_format: Annotated[
+        DataFormat,
+        typer.Option(
+            help="Dataformat to store the output dataset in.",
+        ),
+    ] = DataFormat.WKW,
+    layer_name: Annotated[
+        Optional[str], typer.Option(help="New name for the layer.")
+    ] = None,
+    compress: Annotated[
+        bool, typer.Option(help="Compress the output dataset.")
```

Comment:
```suggestion
        bool, typer.Option(help="Enable compression of the output dataset.")
```

Reply:
In this comment you use the term `output dataset`. Above you use `target dataset`. Please settle on one.